packages/dashmate/README.md

```diff
@@ -226,23 +226,34 @@ The `reset` command removes all data corresponding to the specified config and a
 ```
 USAGE
   $ dashmate reset [--config <value>] [-v] [-h] [-f] [-p]
-
 FLAGS
   -f, --force          skip running services check
   -h, --hard           reset config as well as data
   -p, --platform-only  reset platform data only
   -v, --verbose        use verbose mode for output
   --config=<value>     configuration name to use
-
 DESCRIPTION
   Reset node data
 ```
 
 To reset a node:
 ```bash
 $ dashmate reset
+=======
+#### Hard reset
+$ dashmate reset --hard
+```
+With the hard reset mode enabled, the corresponding config will be reset in addition to the platform data. After a hard reset, it is necessary to run the node [setup](#setup-node) to proceed.
```

Comment:
Just adding a blank line after the close of the code block for readability of the markdown (doesn't affect actual display of file on GitHub).
```suggestion

With the hard reset mode enabled, the corresponding config will be reset in addition to the platform data. After a hard reset, it is necessary to run the node [setup](#setup-node) to proceed.
```